data/scripts/actions/items/garlic_bread.lua

```diff
@@ -5,5 +5,5 @@ function garlicBread.onUse(player, item, fromPosition, target, toPosition, isHot
 	return true
 end
 
-garlicBread:id(9087)
+garlicBread:id(8194)
```

Comment:
fixed in: https://github.com/opentibiabr/canary/commit/5425d1ce212cb4942e298ea731943276a0c3ec35